### ISSUE TITLE
Client window mixer scrollbar

### ISSUE THREAD

**Issue by pljones** [jamulussoftware/jamulus#1579]
**Describe the bug**
The client main window is allowed to get small enough that the mixer panel shows a scrollbar.

**To Reproduce**
* Connect to a server
* Reduce the client window size down:
![image](https://user-images.githubusercontent.com/1549463/116606845-38679d80-a929-11eb-8a4d-ab87812b5663.png)
* Continue reducing the window size down.

**Expected behaviour**
The window should not get smaller.

**Actual behaviour**
The window gets smaller, causing a vertical scrollbar.
![image](https://user-images.githubusercontent.com/1549463/116607039-69e06900-a929-11eb-9222-9ac4aa2d17d0.png)

**Operating system**
Windows

**Version of Jamulus**
Version 3.7.0dev-74f55f2

**Additional context**
I don't remember when it started happening but it was probably in 3.7.0 or shortly before.

**Comment by dcorson-ticino-com** [comment]
I think it has always been like this.
I remember my first use of 2 line mixerboard and not seeing the second line.
I had to scroll to find it.
How do you want to handle moving from 1 line to multi-line if there are no scroll bars ?  Resize the window? and if that isn't enough ?

**Comment by pljones** [comment]
Yeah, it could have been -- I guess until the multiple rows, I'd sort of never bothered about vertical height.  The constraining factor is the left-most pane, currently, so it could have changed when that got adjusted.

I'd actually like the window to open to the minimum size vertically to accommodate the rows used.  Currently, setting to three rows, I can't open the window enough -- I don't know if that's a Jamulus "max" or a Qt "max" or a Windows "max" height, though.  Reducing the number of rows shouldn't reduce the height, though.

I'm thinking the min window height should only accommodate one row plus a _horizontal_ scrollbar in the mixer pane.  If the number of rows increases, like I say, increasing the height automatically would be nice -- but it shouldn't be a constraint.

**Comment by ann0see** [comment]
I think not having a scroll bar would make Android and iOS clients unusable since the display is just too small. This is a broader issue I think and needs a bigger UI redesign?

**Comment by pljones** [comment]
![image](https://github.com/jamulussoftware/jamulus/assets/1549463/226d1550-0a1c-45ba-b65d-847889c0e0f7)
I guess the point is there's still room for the height to be reduced before a scrollbar would be needed.

And yes, the mobile UI needs to be fluid rather than static as the desktop UI is.
